library_management_system_downloader/evaluate_api_response_for_fulltext_indication.py

```diff
@@ -0,0 +1,47 @@
+from bs4 import BeautifulSoup
+import logging
+import unittest
+
+
+def fulltext_indication(api_response_xml):
+    """Given XML from an API response, determine whether fulltext is available
+    for that item. Return True if fulltext is available, and False if it is
+    not."""
+    parseable_response_xml = BeautifulSoup(
+            api_response_xml,
+            features='xml')
+
+    doi_value = parseable_response_xml.find(
+            id='rft.doi')
+
+    full_text_indicator_value = parseable_response_xml.find(
+            id='full_text_indicator')
+
+    logging.info(
+          f'The value for the "full_text_indicator" key for doi "{doi_value}" '
+          'is "full_text_indicator_value".')
+
+    if full_text_indicator_value is None:
+        return 0
+    return int(full_text_indicator_value.string == 'true')  # This will
+    # evaluate to either 1 or 0.
+
+# =============================================================================
+# Function examples / tests
+# =============================================================================
+
+
+class TestFulltextIndication(unittest.TestCase):
```

Comment:
We should use pytest for testing... but we can save that for a later PR.

Reply:
I had a few extra minutes today, and this seemed small, so I added a tests file for the function, in 6f2653. Is that how you prefer it?